weave/trace_server/trace_server_interface.py

```diff
@@ -196,6 +200,18 @@ class CallsQueryStatsRes(BaseModel):
     count: int
 
 
+class CallRenameReq(BaseModel):
+    project_id: str
+    call_id: str
+    # wb_user_id gets generated from auth params
+    wb_user_id: typing.Optional[str] = None
```

Comment:
When is this None? If it is only expected to be None over the wire, then you should consider a specific Req class inside the server definition itself.

Reply:
ah okay, this is the same as Delete so I can do both that way...

Reply:
I realize this is a bit lame but im not 100% sure what you mean, or where the best place for this type to live...

Reply:
ok let's chat... basically:

We should have `class CallRenameReq(BaseModel):` here require the `wb_user_id` and the trace server itself should have a class with such field omitted

Reply:
honestly, it is a small nit, so maybe not worth it. In Jamie's `https://github.com/wandb/weave/pull/1718/files#diff-2628396670e408d29134ac19cc8f01d7d12e9e60821a514e02b0139a03e05d8c`, the id is required actually

Reply:
Right, but doesn't `call_rename` have to be identical in: 
- `remote_http_trace_server.py`
- `clickhouse_trace_server_batched.py`
- `TraceServerInterface` in `trace_server_interface.py`?

Over the wire we _don't_ need `wb_run_id`, but in the other places we do. @tssweeney